### PR TITLE
update WS4Net support to 0.15

### PR DESCRIPTION
**Pull request by uwx**
This adds .NET Standard 1.3+ support on Windows.

### CODE REVIEW

**Comment by NaamloosDT** [comment]
oh wait
i meant to just add emzi

all seems good to me

**Comment by Emzi0767** [comment]
Nope. The project stays at 0.14, and for a reason. We cannot push a live package with 0.15.

**Comment by Emzi0767** [comment]
I'll look later

File: docs/articles/alt_ws.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ you need to use an alternative WebSocket client implementation.
 
 Currently, there are 2 implementations available on NuGet:
 
-* `DSharpPlus.WebSocket.WebSocket4Net`: This implementation is recommended if you're targeting .NET Framework 4.5+, and using 
-  Windows 7 or Mono 4.4.2.
+* `DSharpPlus.WebSocket.WebSocket4Net`: This implementation is recommended if you're targeting .NET Framework 4.5+ or [.NET
```
Comment (Emzi0767) [inline docs/articles/alt_ws.md]: Should be reverted, since the impl should be split anyways.

**Pull request merged**